go/conformance/complete_test.go

```
@@ -0,0 +1,138 @@
+// completeWord is the reference's answer for a partial word at a command path.
+//
+// Only the lines the CLI itself produced: where the reference has nothing to
+// offer it falls back to listing the working directory, and those lines describe
+// the machine rather than the spec. The Go side says `\x01files` there instead,
+// which is compared separately below.
+func completeWord(t *testing.T, usageBin, kdl string, words []string, cword int) []string {
+	t.Helper()
+	args := []string{"complete-word", "--shell", "bash", "-f", kdl,
+		"--cword", itoa(cword), "--"}
+	args = append(args, words...)
+	out, err := exec.Command(usageBin, args...).Output()
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)